boot/bootchain.go

```diff
@@ -47,7 +47,7 @@ type bootChain struct {
 	KernelRevision string   `json:"kernel-revision"`
 	KernelCmdlines []string `json:"kernel-cmdlines"`
 
-	model          *modeenvModelForSealing
+	model          modeenvModelForSealing
```

Comment:
looking at things more I'm not sure we need to stick an extra model here anymore because bootChain itself has enough values to build an object that supports secboot.ModelForSealing, no?